app/views/partials/_navbar.html.erb

```diff
@@ -23,7 +23,9 @@
         </ul>
         <ul class="navbar-nav ml-auto">
             <li class="nav-item">
-                <a class="glyphicon glyphicon-shopping-cart" href="" id=order-icon> Cart</a>
+              <% if session[:order_id] %>
+                <%= link_to ' Checkout', order_path(session[:order_id]), class: 'glyphicon glyphicon-shopping-cart', id: "order-icon" %>
```

Comment:
Here you are sending the order as params so please use that in your controller

Reply:
Yes, we'll fix that

Reply:
Fixed

Reply:
Why the space before `' Checkout'` ?

Reply:
Fixed it. Apparently was used as a "duct tape" solution to put a space between the cart icon and the text. We'll do it with css instead